angular.json

```diff
@@ -25,7 +25,7 @@
               "src/assets"
             ],
             "styles": [
-              "@angular/material/prebuilt-themes/indigo-pink.css",
+              "@angular/material/prebuilt-themes/deeppurple-amber.css",
```

Comment:
I think we should keep the indigo-pink theme from master... Looks like you did your slack-n-tell right in your follow-along demo branch and made a different theme choice...